README.md

```diff
@@ -100,16 +141,20 @@ augmentation_pipeline = AugmentationSequential(
     RandomAffine((-45., 45.), p=1.),
     RandomBrightness((0.,1.), p=1.)
 )
+# Leveraging StableDiffusion models
```

Comment:
can we show above loading images with kornia_rs ?

also I'm not very fond of promoting augmentations as everyone thinks we are only an augmentations library in the end

Reply:
Can you modify it directly there? To add more examples or anything else. What I wanted to say here is that we support many different input types and the easy save and visualization.

I think this Quick Start is to promote the use cases in practic. We can include `kornia_rs` as a torch tensor example maybe.

Reply:
added

Reply:
We may add a kornia.model example here once the module is ready.

Reply:
yep, I vote for RTDETR @cjpurackal